e2e/tests/omnibus-base-goerli-andromeda.toml/Perps_Trading.e2e.js

```diff
@@ -271,23 +277,19 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
   it('should open a 0.1 btc position', async () => {
     const btcMarketId = 200;
     const marketId = btcMarketId;
+    const strategyId = 0;
     const settlementStrategyId = extras.btc_pyth_settlement_strategy;
-    const strategy = await PerpsMarketProxy.getSettlementStrategy(marketId, 0);
+    const strategy = await PerpsMarketProxy.getSettlementStrategy(marketId, strategyId);
 
-    const {
-      settlementDelay: settlementDelayBn,
-      feedId: priceFeedId,
-      priceVerificationContract,
-      commitmentPriceDelay,
-    } = strategy;
+    const { feedId: priceFeedId, priceVerificationContract, commitmentPriceDelay } = strategy;
 
-    const settlementDelay = settlementDelayBn.toNumber();
     const price = await getPrice(priceFeedId);
     const pythPrice = price.getPriceAsNumberUnchecked();
     const sizeDelta = ethers.utils.parseEther('0.1');
 
     let commitReceipt;
     try {
+      await setSettlementDelay({ strategyId, marketId, delay: 0 });
```

Comment:
I'm not sure that delay 0 is allowed in contracts?

Reply:
The test opened and closed orders without problems

Reply:
```ts
strategy.settlementDelay = strategy.settlementDelay == 0 ? 1 : strategy.settlementDelay;
```
in the code.
so test pass only because it takes longer than 1s.
Let's set it explicitly to 1s and add 1s delay as there is no way around the contract logic.

Reply:
Change to 1 and added a 1s delay

Reply:
Thanks!